admin_guide/manage_users.adoc

```diff
@@ -98,7 +85,7 @@ To delete a user, all three sources of the user must be removed:
 Once you complete these steps, a new account will be created in OpenShift
 when the user logs in again.
 
-==== Deleting the Identification Manager Entry
+. Delete the identification manager entry
```

Comment:
See http://asciidoctor.org/docs/asciidoc-writers-guide/#complex-list-content for how to use `+` to string sentences/paragraphs together so that they start part of the same list item. This way you won't have to do the `[start=#]` thing to keep the numbering right.